foxy/ci-nightly-performance.yaml

```diff
@@ -105,6 +105,11 @@ targets:
 type: ci-build
 underlay_from_ci_jobs:
 - nightly-extra-rmw-release
+keep_files:
```

Comment:
```suggestion
archive_files:
```

Reply:
Done in https://github.com/ros2/ros_buildfarm_config/pull/92/commits/de5100df0fa9e4461bfd3fbf80386c45df4c6dae